Code/Framework/AzToolsFramework/AzToolsFramework/Manipulators/EditorVertexSelection.h

```diff
@@ -20,6 +21,14 @@
 
 namespace AzToolsFramework
 {
+    //! Registers the Actions provided by the EditorVertexSelection while it is active to the Action Manager.
+    //! e.g. Vertex deletion, duplication etc.
+    struct EditorVertexSelection
```

Comment:
suggestion: It might be better to call this something Action related, so maybe `EditorVertexSelectionActions`?